app/src/main/java/io/pslab/communication/ScienceLab.java

```diff
@@ -1612,6 +1837,14 @@ else if ("falling".equals(edgeType2))
         return retData;
     }
 
+    /**
+     * Start logging timestamps of rising/falling edges on ID1
```

Comment:
@neel1998 there is duplicate on line 1850. I think we should keep it here and remove from 1850 since documentation should contain the info of channel.